common/openapi.yml

```diff
@@ -158,6 +158,23 @@ paths:
         '500':
           $ref: '#/components/responses/ServerError'
 
+  /api/batch/v1/runs/{runId}/state:
+    parameters:
+      - $ref: '#/components/parameters/RunId'
```

Comment:
I think `runId` would be taken care of by the `workflow-id` in the request body. It would also be nice to have a stable endpoint to pass into Cromwell config rather than needing to pass in an override each time a workflow is submitted.

Reply:
Correct, I got carried away by implementing the inner logic. Will update PR soon. :)